packages/client/src/generation/utils.ts

```diff
@@ -366,6 +366,14 @@ export function unique<T>(arr: T[]): T[] {
 
   return result
 }
+
+/**
+ * Replaces \ to /
+ */
+export const fixPath = (path: string) => {
+  return path.replace(/\\/g, '/')
```

Comment:
It's an edge case, but on Linux and macOS files and directories may intentionally have backslashes in their names, and it would be incorrect to replace those with path separators.